src/vue-wrapper.ts

```diff
@@ -109,32 +103,35 @@ export class VueWrapper<T extends ComponentPublicInstance>
   get<T extends Element>(selector: string): DOMWrapper<T>
   get(selector: string): DOMWrapper<Element> {
     const result = this.find(selector)
-    if (result instanceof ErrorWrapper) {
-      throw new Error(`Unable to get ${selector} within: ${this.html()}`)
+    if (result instanceof DOMWrapper) {
+      return result
     }
 
-    return result
+    throw new Error(`Unable to get ${selector} within: ${this.html()}`)
   }
 
   findComponent<T extends ComponentPublicInstance>(
     selector: new () => T
-  ): VueWrapper<T> | ErrorWrapper
+  ): VueWrapper<T>
   findComponent<T extends ComponentPublicInstance>(
     selector: FindComponentSelector
-  ): VueWrapper<T> | ErrorWrapper
+  ): VueWrapper<T>
   findComponent<T extends ComponentPublicInstance>(
     selector: any
-  ): VueWrapper<T> | ErrorWrapper {
+  ): VueWrapper<T> {
     if (typeof selector === 'object' && 'ref' in selector) {
       const result = this.vm.$refs[selector.ref]
-      return result
-        ? createWrapper(null, result as T)
-        : new ErrorWrapper({ selector })
+      if (result) {
+        return createWrapper(null, result as T)
+      }
     }
 
     const result = find(this.vm.$.subTree, selector)
-    if (!result.length) return new ErrorWrapper({ selector })
-    return createWrapper(null, result[0])
+    if (result.length) {
+      return createWrapper(null, result[0])
+    }
+
+    return createWrapperError('VueWrapper')
```

Comment:
I see we lost the selector on the error wrapper. What was it used for?

Reply:
Turns out it wasn't used for anything - maybe it was previously, but in the current master is it not used. So I just deleted it.